galaxykubeman/values.yaml

```diff
@@ -252,6 +263,14 @@ galaxy:
     galaxy.yml:
       galaxy:
         enable_beta_gdpr: true
+    job_conf.yml:
+        execution:
+          environments:
+            tpv_dispatcher:
+              tpv_config_files:
+                - https://raw.githubusercontent.com/galaxyproject/tpv-shared-database/main/tools.yml
+                - lib/galaxy/jobs/rules/tpv_rules_local.yml
```

Comment:
This line can be removed?

Reply:
Turns out this is a required line. Otherwise, no destinations are available for jobs. 
https://github.com/galaxyproject/galaxy-helm/blob/c3128f1bbf1e539d816a73926fe34dca4f4cb8fb/galaxy/values.yaml#L503